### ISSUE TITLE
"Commits in Last Push" not shown in test report

### ISSUE THREAD

**Issue by rhaas80** [EinsteinToolkit/tests#45]
The table "Commits in Last Push" does not show any commits anymore. 

The list of commits is present in https://github.com/EinsteinToolkit/tests/tree/gh-pages/records/version_434 then gone in https://github.com/EinsteinToolkit/tests/tree/gh-pages/records/version_435 and present again in https://github.com/EinsteinToolkit/tests/tree/gh-pages/records/version_436 but mostly missing after that (it is present in https://github.com/EinsteinToolkit/tests/tree/gh-pages/records/version_452 https://github.com/EinsteinToolkit/tests/tree/gh-pages/records/version_470 https://github.com/EinsteinToolkit/tests/tree/gh-pages/records/version_471 https://github.com/EinsteinToolkit/tests/tree/gh-pages/records/version_476 https://github.com/EinsteinToolkit/tests/tree/gh-pages/records/version_490 https://github.com/EinsteinToolkit/tests/tree/gh-pages/records/version_523 https://einsteintoolkit.github.io/tests/build_526.html https://github.com/EinsteinToolkit/tests/tree/gh-pages/records/version_535 https://github.com/EinsteinToolkit/tests/tree/gh-pages/records/version_542 https://github.com/EinsteinToolkit/tests/tree/gh-pages/records/version_551).

Also since about December 05 the tests seem to run much more frequently than before. So possibly they get triggered by things other than pushes to the code part of the repo?

**Comment by rhaas80** [comment]
For example commit https://github.com/EinsteinToolkit/tests/commit/6afdf37b1f9d2583bf7ee355eee93dbbb198f49f shows that there were changes to submodules

```
 updated submodules

Submodule repos/nrpytutorial 51433fa..56f3784:
  > Move Tutorial-NS_radial_oscillations.ipynb to in_progress, as this notebook was never quite completed.
  > Move deprecated Sommerfeld directory to Deprecated. Also replace last remaining link to deprecated Sommerfeld notebook with new CurviBC notebook
  > README.md: Add GitHub Actions CI badges for all workflows
```

and triggered CI run https://github.com/EinsteinToolkit/tests/actions/runs/3803437310 which has a failure reported in the "Check for changes" step:

```
   cd gh-pages
  git fetch origin # fetch changes from origin remote
  if ! git diff --quiet master..origin/master ; then
    echo "CHANGES_MADE=True" >> $GITHUB_ENV
  fi
  shell: /usr/bin/bash -e {0}
  env:
    CHANGES_MADE: False
    TESTS_FAILED: False
    pythonLocation: /opt/hostedtoolcache/Python/3.11.1/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.11.1/x64/lib
fatal: ambiguous argument 'master..origin/master': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
```

and pushed gh-pages commit cb1cd971e5 https://github.com/EinsteinToolkit/tests/commit/cb1cd971e5fe1a85b76acabed0789a98c07e5739 producing build_574.html which exists multiple times (!) in test_nums.csv and contains no list of changes.

**Comment by rhaas80** [comment]
I believe this has been fixed in  git hash \[2778c88cd1\](https://github.com/EinsteinToolkit/tests/commits/2778c88cd1028d59f7fa41d066cb0077f840ff1d) "logpage: pass directory location to store module functions" of \[tests\](https://github.com/EinsteinToolkit/tests)
